### DISCUSSION TITLE
Déplacer les liens du pieds de page dans l'entête

### DISCUSSION THREAD

**Discussion by ggrossetie** [EcrituresNumeriques/stylo#1095] [Ideas]
L'objectif premier serait de supprimer la barre de scroll verticale sur la page de l'éditeur de texte. L'éditeur de texte (Monaco) est configuré pour prendre toute la hauteur (restante) de la fenêtre du navigateur:

![image](https://github.com/user-attachments/assets/9084a28a-8fc8-456f-b246-8b1e35218129)

La barre de scroll verticale de la fenêtre sert uniquement à afficher le pied de page : 

![image](https://github.com/user-attachments/assets/cadb244e-7461-4b32-a5eb-fc0048bf7683)

Peut être qu'on pourrait transformer le lien Documentation en un menu "Aide" (je n'ai pas trouvé mieux pour le moment) qui affiche : 

- La version de Stylo
- Le lien vers la documentation (lien externe)
- Le journal des modification (lien externe)
- Lien vers la page confidentialité

Est-ce qu'on déplace la case à cocher "J'accepte de partager mes statistiques de navigation" dans la page confidentialité ?

![image](https://github.com/user-attachments/assets/b2b8c351-c2ca-4c65-948e-2c67a3656660)

**Comment by RochDLY** [comment]
Est-ce que pour la version de Stylo ça ferait sensde le mettre sous le logo de stylo à gauche ?

**Comment by RochDLY** [comment]
Pour le reste ça me semble un bonne idée!

**Comment by ggrossetie** [comment]
On a décidé de supprimer le pied de page sur la page de l'éditeur de texte (i.e., quand l'éditeur de texte prend la hauteur de la fenêtre)
